model/godleyTab.h

```diff
@@ -0,0 +1,36 @@
+/*
+  @copyright Steve Keen 2020
+  @author Russell Standish
+  @author Wynand Dednam
+  This file is part of Minsky.
+
+  Minsky is free software: you can redistribute it and/or modify it
+  under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  Minsky is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with Minsky.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef GODLEYTAB_H
+#define GODLEYTAB_H
+#include <itemTab.h>
+
+namespace minsky
+{
+	 
+  class GodleyTab: public ItemTab
+  {	  
+  public:
+    bool itemSelector(ItemPtr i) override {if (auto g=dynamic_cast<GodleyIcon*>(i.get())) {return g->editorMode();}; return false;}    
```

Comment:
Wouldn't we want the godley table displayed always, not just when they're in "editor mode"?

Reply:
Yes, that's true. I'll change it. I also forgot to assign the itemCoords object using the [] operator instead of emplace...